docs/src/pages/guides/styling.md

```diff
@@ -206,13 +206,12 @@ Astro can be configured to use [Tailwind][tailwind] easily! Install the dependen
 npm install --save-dev tailwindcss
 ```
 
-And create 2 files in your project root: `tailwind.config.cjs` and `postcss.config.cjs`:
+And create 2 files in your project root: `tailwind.config.js` and `postcss.config.js`:
```

Comment:
Nit: Being explicit that this is a `cjs` config file will help users using `"type": "module"`.

Unless Tailwind has removed support for `cjs` config files? Otherwise I'd leave this as is.

Reply:
can we say it in the docs, that for users using "type": "module" use .cjs extension

Reply:
Not trying to be stubborn about this, but if users are familiar with the semantics of CJS/ESM in Node, they will know enough to change the extension on their own.

The `.cjs` extension is the least error-prone, especially for novice users who might just copy + paste.

You'll notice that we always document the `astro.config.mjs` file for the same reason. You _can_ use `astro.config.js` with `"type": "module"`, but we don't want to get into Node's CJS/ESM behavior every time we mention the config file.

Reply:
👍 understood, I'll update it now